cmd/utils.go

```diff
@@ -483,3 +483,14 @@ func reverseStringSlice(input []string) {
 		input[left], input[right] = input[right], input[left]
 	}
 }
+
+// Returns the mode in which MinIO is running
+func getMinioMode() string {
```

Comment:
This function needs to get modes for Gateway also. `Miniomode` is also used in `Gateway` see https://github.com/minio/minio/blob/master/cmd/gateway-main.go#L301

Reply:
@kannappanr `getMinioMode` is not used by gateway. The mode string is simpler to derive in the case of gateway.